README.md

```diff
@@ -1 +1,69 @@
-# week2Milestone1
\ No newline at end of file
+# Responsive Design Milestone One
+
+This Is the Responsive design project for microverse 2.0 curriculum. in this project we have used HTML and CSS to make a clone of a fully responsive webpage given in the following link. https://perma.cc/M5ZV-Q2D6
+
+I am using float grid and flex and any options I choose to layout and style the options to appear like on the page and we have used media queries to display for mobile view 
+
+
+![image](https://user-images.githubusercontent.com/70282171/108868739-2e5eb100-75ff-11eb-8cff-26e77cac16fc.png)
+
+
+## Built With
+
+- HTML and CSS
+
+## Live Demo
+
+[Live Demo Link] [MilestoneOne](https://stephanie041996.github.io/week2Milestone1/)
+
+
+## Authors
+
+👤 **Author1**
+GitHub: @stephanie041996
+LinkedIn: [linkedin](https://www.linkedin.com/in/stephanie-sakuhuni-a81029140/)
+👤 Author Stephanie Sakuhuni
+
+👤 **Author2**
+GitHub: [@Solenka1](https://github.com/Solenka1)
+Twitter: [@GorgiMitev](https://twitter.com/GorgiMitev)
+LinkedIn: [gorgi-mitev](https://www.linkedin.com/in/gorgi-mitev-a350311b8/)
+👤 Author Gorgi Mitev
+
+## How to Download and copy the project
+###Steps by download
```

Comment:
- Give it some space to properly format. `### Steps by download`. This sentence should be something like `### Steps to download`. It makes more sense grammatically.